miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/analytics/Event.kt

```diff
@@ -0,0 +1,7 @@
+package com.rakuten.tech.mobile.miniapp.analytics
```

Comment:
I think we should instead create a separate analytics module for `com.rakuten.tech.mobile.analytics` (like mentioned earlier).

We can't release our SDK as it is like this because analytic events will not be sent even if the Host App includes analytics (we don't yet know when public analytics will be available). But using a separate `compileOnly` module will let Host Apps use analytics immediately, and it will still be easy to add the real analytics to the demo app later.

Also, using an "undefined" package of `com.rakuten.tech.mobile.analytics` seems more generic and could let a developer define their own implementation of analytics instead of needing to rely on the dependency we provide.